README.md

```diff
@@ -217,11 +217,15 @@ However, we emphasize that these ports are by developers outside the
 libphonenumber project. We do not evaluate their quality or influence their
 maintenance processes.
 
-*   Android: Java version of libphonenumber can be used on Android as is.
-    https://github.com/MichaelRocks/libphonenumber-android is a repackaged
-    port optimized for Android by using `AssetManager#open()` instead of
-    `Class#getResourcesAsStream()` to load metadata. See [FAQ](https://github.com/googlei18n/libphonenumber/blob/master/FAQ.md#optimize-loads)
-    for why this is better
+*   Android-optimized: If you don't want to use our Java version, which loads
+    the metadata from `Class#getResourcesAsStream` and asks that Android apps
+    follow the Android loading best practices of repackaging the metadata and
+    loading from `AssetManager#open()` themselves, as documented in our
```

Comment:
I think you could just remove "as documented in our" and just put "FAQ" in parenthesizes.